src/server.js

```diff
@@ -138,20 +144,30 @@ let load = loader({
       // Create server and start listening
       let server = http.createServer(app);
       await new Promise((accept, reject) => {
-        server.listen(cfg.server.port, accept);
+        server.listen(cfg.server.port);
+        server.once('listening', () => {
+          console.log("Listening on port: " + cfg.server.port);
+        });
         server.once('error', reject);
+        server.once('close', accept);
```

Comment:
Why not accept once listening?

I think the test cases relies on this? At least that's how I build the `helper.js` in other projects... 
Oh, this project doesn't have tests... obviously, since it's just a tiny bit of UI...